src/app/ReadPrepareParams.h

```diff
@@ -48,6 +48,7 @@ struct ReadPrepareParams
     size_t mAttributePathParamsListSize             = 0;
     DataVersionFilter * mpDataVersionFilterList     = nullptr;
     size_t mDataVersionFilterListSize               = 0;
+    bool mEnableCachedDataVersionFilter             = false;
```

Comment:
This is never read.  Why is it here?

Reply:
Accidentally put it here, never used. removed, thanks